### PR TITLE
background color saves in database

### PR DESCRIPTION
**Pull request by christinaashworth**
hallelujah

How to test: 
1. run branch
2. select a background color
3. exit app
4. run again
5. pray to God it has saved the color
6. at least two ritual sacrifices
7. check to see if color is the one you chose before exiting program

### CODE REVIEW

**Review by rryb77 — APPROVED** [review]
👍

**Comment by rickiele** [comment]
👍

File: TabloidCLI/Models/BGColor.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
```
Comment (rickiele) [inline TabloidCLI/Models/BGColor.cs:4]: Hello comment

**Review by rickiele — COMMENTED** [review]
a comment

**Pull request merged**